test/deno/web_test.ts

```diff
@@ -1,17 +1,17 @@
 import { sign, verify } from "../../pkg/dist-web/index.js";
```

Comment:
It seems this import is out of date now.

Reply:
for browser we need to build it once

Reply:
The `dist-web` folder doesn't exist anymore. It should be `dist-bundle`